tests/test_plugin.py: TypeError: missing 1 required positional argument

Running the project's unit tests, the following two tests fail due to missing positional arguments in the unit test: 

```
=========================== short test summary info ============================
FAILED tests/test_plugin.py::test_pylsp_format_document - TypeError: pylsp_format_document() missing 1 required positional argument: 'document'
FAILED tests/test_plugin.py::test_pylsp_format_range - TypeError: pylsp_format_range() missing 1 required positional argument: 'range'
```

The function signatures were changed in 66fd3a4780efd9fc3a0002dbdf4a0b4febcac4cd and 2867a45eca0362390d54b184fbf519c7ac268a52, respectively, but the unit tests haven't been adapted to the new signature.

--- a/tests/test_plugin.py
+++ b/tests/test_plugin.py
@@ -108,8 +108,12 @@ def test_pylsp_settings(config):
     assert plugins["isort"] not in config.disabled_plugins
 
 
-def test_pylsp_format_document(config, unformatted_document, formatted_document):
-    actual = _receive(plugin.pylsp_format_document, config, unformatted_document)
+def test_pylsp_format_document(
+    config, workspace, unformatted_document, formatted_document
+):
+    actual = _receive(
+        plugin.pylsp_format_document, config, workspace, unformatted_document
+    )
 
     text = _read_content(formatted_document.path)
     range = plugin.Range(
@@ -121,12 +125,14 @@ def test_pylsp_format_document(config, unformatted_document, formatted_document)
     assert actual == expected
 
 
-def test_pylsp_format_range(config, unformatted_document):
+def test_pylsp_format_range(config, workspace, unformatted_document):
     range = plugin.Range(
         start={"line": 2, "character": 0},
         end={"line": 9, "character": 0},
     )
-    actual = _receive(plugin.pylsp_format_range, config, unformatted_document, range)
+    actual = _receive(
+        plugin.pylsp_format_range, config, workspace, unformatted_document, range
+    )
 
     text = "\n".join(
         [

